src/drc/api/permissions.py

```diff
@@ -20,3 +20,19 @@ class InformationObjectRelatedAuthScopesRequired(RelatedObjAuthScopesRequired):
     """
     permission_fields = ('informatieobjecttype', 'vertrouwelijkheidaanduiding')
     obj_path = 'informatieobject'
+
+    # Define the property of the ForeignKey of which the permission fields will
+    # be checked
+    obj_property = 'latest_version'
```

Comment:
seeing as this results in just another `getattr` call, doesn't it make more sense to specify:

```python
obj_path = 'informatieobject.latest_version'
```
?